### PR TITLE
Make sure we close the streams when reading a file

### PR DESCRIPTION
**Pull request by vojtechtrefny**
Python's garbage collector should close these automatically, but we shouldn't depend on that and close the file descriptor when no longer needed.

### CODE REVIEW

File: blivet/util.py [hunk 1]
```diff
@@ -280,17 +280,17 @@ def get_mount_paths(dev):
...
+                (device, path, _rest) = mnt.split(None, 2)
+            except ValueError:
+                continue
 
-        if path == mountpoint:
-            mount_device = device
-            break
+            if path == mountpoint:
+                mount_device = device
+                break
```
Comment (VladimirSlavik) [inline blivet/util.py:293]: I would expect this fix to be a separate commit.

**Review by VladimirSlavik — APPROVED** [review]
LGTM, thanks.

Not a team member of course.

File: blivet/util.py (hunk 1 above)
Comment (vojtechtrefny) [reply blivet/util.py:293]: This isn't a separate change, I am just using the `open` contextmanager here so I needed to indent this block. I guess it's just git diff showing it in a weird way.
